package.json

```diff
@@ -21,7 +22,8 @@
     "lint": "eslint --ext js,jsx,ts,tsx packages && stylelint '**/*.{js,jsx,ts,tsx}'",
     "publish-npm": "lerna run --scope @razorpay/blade publish-npm",
     "release": "lerna run --scope @razorpay/blade generate-github-npmrc && changeset publish",
-    "build": "lerna run --scope @razorpay/blade build"
+    "build": "lerna run --scope @razorpay/blade build",
+    "postinstall": "yarn workspace eslint-plugin-blade build"
```

Comment:
why can't we use lerna here? We're using lerna in above scripts, lets do the same thing everywhere?

Reply:
@chaitanyadeorukhkar We can use Lerna here, I've also updated the `tsc:blade` script to use lerna for consistency.